app/controllers/dashboard_controller.rb

```diff
@@ -10,9 +10,9 @@ def show
     # Also, it processes all of a user's works, but not all may be rendered on the dashboard.
     @draft_works = []
     @status_map = current_user.works.to_h do |work|
-      status = work.druid ? Sdr::Repository.status(druid: work.druid) : nil
-      @draft_works << work if status&.open?
-      [work.id, StatusPresenter.new(status: status)]
+      version_status = work.druid ? Sdr::Repository.status(druid: work.druid) : VersionStatus::NilStatus.new
```

Comment:
Using a special NilStatus object when the object is not yet registered instead of handling that special case within VersionStatus.

Reply:
@justinlittman What would you think about having `Sdr::Repository` being responsible for handing back the `NilStatus` instead of the controller switching on what it gets back?

Reply:
That's what I originally did, but realized that in every case but this one, no druid is an error (and Sdr::Repository.status detects and raises on).

That being said, open to discussion.